editor/gui/editor_run_bar.h

```diff
@@ -72,8 +74,13 @@ class EditorRunBar : public MarginContainer {
 	EditorRun editor_run;
 	EditorRunNative *run_native = nullptr;
 
+	enum MovieMakerMenuItem {
+		TOGGLE_MOVIE_MAKER,
+		OPEN_MOVIE_MAKER_SETTINGS
```

Comment:
```suggestion
		MOVIE_MAKER_TOGGLE,
		MOVIE_MAKER_OPEN_SETTINGS
```
Due to how enums work in C++ (the values are available without using the enum name), we usually prefix them to avoid potential conflicts and better organize the values.